map_gui/src/tools/minimap.rs

```diff
@@ -101,38 +101,35 @@ impl<A: AppLike + 'static, T: MinimapControls<A>> Minimap<A, T> {
         let zoom_col = {
             let mut col = vec![ctx
                 .style()
-                .btn_plain_icon("system/assets/speed/speed_up.svg")
+                .btn_plain_icon("system/assets/speed/plus.svg")
```

Comment:
just renamed the file - presumably a legacy name, but it's not currently used anywhere else.